projects/envoyproxy/envoy/Makefile

```diff
@@ -0,0 +1,29 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+
+ENVOY_IMAGE=public.ecr.aws/appmesh/aws-appmesh-envoy:$(GIT_TAG)
+IMAGE_BUILD_ARGS=ENVOY_IMAGE
+
+REPO=envoy
+REPO_OWNER=envoyproxy
+
+IMAGE_COMPONENT=envoy
+IMAGE_NAMES=$(IMAGE_COMPONENT)
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base-glibc
+
+REPO_NO_CLONE=true
+SIMPLE_CREATE_BINARIES=false
+
+BUILD_TARGETS=local-images
```

Comment:
Let's avoid this. It may require setting some additional vars but I prefer not to override build and release.

Reply:
turns out these are not even needed